### PR TITLE
IESG review update

### PR DESCRIPTION
**Pull request by vb-balazs**
Fixing the comments received during the IESG review.

### CODE REVIEW

**Comment by vb-balazs** [comment]
MPLS document

File: mpls/draft-ietf-detnet-mpls.xml [hunk 1]
```diff
@@ -505,6 +506,9 @@ DFx = DetNet member flow x over a TE LSP
    Word (PWMCW) defined in <xref target="RFC4385"/>. The d-CW formatted
    as shown in <xref target="fig_detnet_cw"/> MUST be present in all
    DetNet packets containing app-flow data.
+   This format of the d-CW was created in order (1) to allow larger S/N 
+   space to avoid S/N rollover frequency in some applications and (2) to 
+   allow non-skip zero S/N what simplifies implementation."
```
Comment (louberger) [inline mpls/draft-ietf-detnet-mpls.xml]: don't think this should be added -- see list discussion

Comment (vb-balazs) [inline mpls/draft-ietf-detnet-mpls.xml]: Remove added text. Lou commented on the list

File: mpls/draft-ietf-detnet-mpls.xml [hunk 2]
```diff
@@ -885,6 +889,8 @@ DFx = DetNet member flow x over a TE LSP
     description of the Associated Channel mechanism, and to the
     DetNet work on OAM for more information DetNet OAM.
    </t>
+   <t> Additional considerations on DetNet specific OAM are further study.
+   </t>
```
Comment (vb-balazs) [inline mpls/draft-ietf-detnet-mpls.xml:894]: DetNet-specific OAM are subjects for further study.

**Review by louberger — DISMISSED** [review]
comments need to be addressed...

**Comment by vb-balazs** [comment]
MPLS: All 01-27 changes updated

File: mpls/draft-ietf-detnet-mpls.xml (hunk 2 above)
Comment (louberger) [reply mpls/draft-ietf-detnet-mpls.xml:894]: are you still planning on making this change?

File: mpls/draft-ietf-detnet-mpls.xml (hunk 1 above)
Comment (louberger) [reply mpls/draft-ietf-detnet-mpls.xml]: keep this text - per list

Comment (vb-balazs) [reply mpls/draft-ietf-detnet-mpls.xml]: No remove. See list discussions.

**Review by jfarkas — APPROVED** [review]

**Review by jfarkas — APPROVED** [review]

**Comment by vb-balazs** [comment]
> comments need to be addressed...

Done

**Pull request merged**